### PR TITLE
fix(chart): search api exposed directly with ALB

### PR DESCRIPTION
**Pull request by rtrompier**

### CODE REVIEW

File: chart/env/prod.yaml [hunk 1]
```diff
@@ -389,6 +390,12 @@ search:
   replicas: 7
   service:
     type: NodePort
+  ingress:
+    enabled: true
+    annotations:
+      alb.ingress.kubernetes.io/group.order: "1"
+      alb.ingress.kubernetes.io/target-node-labels: role-datasets-server-search=true
+      alb.ingress.kubernetes.io/target-group-attributes: stickiness.enabled=true,stickiness.lb_cookie.duration_seconds=300
```
Comment (severo) [inline chart/env/prod.yaml:398]: ok, but I don't think we pass cookies anyway

File: chart/env/prod.yaml [hunk 2]
```diff
@@ -389,6 +390,12 @@ search:
   replicas: 7
   service:
     type: NodePort
+  ingress:
+    enabled: true
+    annotations:
+      alb.ingress.kubernetes.io/group.order: "1"
```
Comment (severo) [inline chart/env/prod.yaml:396]: OK! this was the missing part, right?

**Review by severo — APPROVED** [review]
OK!

Tbh, I didn't understand the manipulation on `datasetsServer.instance.ingress.annotations`

**Review by AndreaFrancis — APPROVED** [review]
Sorry, I can't give helpful feedback but it looks good overall.

File: chart/env/prod.yaml (hunk 2 above)
Comment (rtrompier) [reply chart/env/prod.yaml:396]: Yep

File: chart/env/prod.yaml (hunk 1 above)
Comment (rtrompier) [reply chart/env/prod.yaml:398]: let's try ;) 
The LB should create a cookie during the first call to remember the instance. During the 2nd call, the ALB will read the cookie, and redirect to the same instance.

Comment (severo) [reply chart/env/prod.yaml:398]: Ah ok!

**Pull request merged**